### ISSUE TITLE
[IDEA] Jackpot and Gravitational swapping

### ISSUE THREAD

**Issue by Heyo7726** [BuilderDolphin/dolphSol-Macro#67]
Basically, when its not on your bonus roll, the macro automatically equips Jackpot, but when you are on your bonus roll it automatically equips gravitational for bonus luck.

**Comment by v8f** [comment]
## This is almost impossible to do and most likely will not ever happen.
### Here is why.
So basically what you are asking for, is bonus roll detection. If you think the auras are detected by text or something, for example if you set up a Discord server for the webhook it sends aura images but that is detected by color. The autohotkeyscript will most likely need an image for reference, like "bonus roll ready", but it is near impossible and very hard to do. just telling you for a heads up!:)

**Comment by Heyo7726** [comment]
> ## This is almost impossible to do and most likely will not ever happen.
> ### Here is why.
> So basically what you are asking for, is bonus roll detection. If you think the auras are detected by text or something, for example if you set up a Discord server for the webhook it sends aura images but that is detected by color. The autohotkeyscript will most likely need an image for reference, like "bonus roll ready", but it is near impossible and very hard to do. just telling you for a heads up!:)

Yeah, idk if its possible but something to count up how many times you the roll sound played or the pixels of "9/10"

**Comment by v8f** [comment]
> > ## This is almost impossible to do and most likely will not ever happen.
> > ### Here is why.
> > So basically what you are asking for, is bonus roll detection. If you think the auras are detected by text or something, for example if you set up a Discord server for the webhook it sends aura images but that is detected by color. The autohotkeyscript will most likely need an image for reference, like "bonus roll ready", but it is near impossible and very hard to do. just telling you for a heads up!:)
> 
> Yeah, idk if its possible but something to count up how many times you the roll sound played or the pixels of "9/10"

Yeah true, the thing is that people can disable UI sfx, or the sound might not detected fast enough.. and also another thing is that gauntlets or devices that make the rolling faster

**Comment by Heyo7726** [comment]
> > > ## This is almost impossible to do and most likely will not ever happen.
> > > ### Here is why.
> > > So basically what you are asking for, is bonus roll detection. If you think the auras are detected by text or something, for example if you set up a Discord server for the webhook it sends aura images but that is detected by color. The autohotkeyscript will most likely need an image for reference, like "bonus roll ready", but it is near impossible and very hard to do. just telling you for a heads up!:)
> > 
> > 
> > Yeah, idk if its possible but something to count up how many times you the roll sound played or the pixels of "9/10"
> 
> Yeah true, the thing is that people can disable UI sfx, or the sound might not detected fast enough.. and also another thing is that gauntlets or devices that make the rolling faster

would take alot but a calculator where you put ur rolling speed and it equips the gauntlets every estimated second. Issue will be the ping.

**Comment by Dusty1234567890** [comment]
For Autohotkey I know it is impossible but of course this is possible to do with lua I can write a script that can detects the counter that when is 9 out of 10 then with Virtual input manager  service we can send a keyevent on the screen ( which is the biggest problem here since screen's size is different on other devices until you set the offset to zero and use percentage value instead,  so with the virtual input manager we simulate a keypress event on the Autoroll button to turn it off then we set the visibility of inventory to true then we set the visibility of grav device to true then we send another keypress event on the equip 's button position,  but I can't say it works perfectly without any misses.  And I'm sure some of people thinks using executers are dangerous so in 1 word I wanted to say it is possibly with lua but not with Autohotkey
